app/lib/formatter.rb

```diff
@@ -53,11 +53,11 @@ def plaintext(status)
 
   def simplified_format(account, **options)
```

Comment:
Is this used by serialization?? If so, we should add an option to determine whether to use `note` (which should be used for federation) or `summary` (which should be used internally).